README.md

```diff
@@ -12,7 +12,8 @@ If you would like to be notified of future conferences that would be added to th
 
 | Conference | Date | Venue | Description |
 |------------|------|-------|-------------|
-|[Google Developer Days](https://developers.google.com/events/gdd-india/) | 1-2 Dec | Bengaluru | Google Developer Days (GDD) are global events showcasing the latest developer products and platforms from Google to help you quickly develop high quality apps, grow & retain an active user base, and tap into tools to earn more. |
+| [Global Blockchain Conference](http://blockchaincon.io) | 16-17 Dec | New Delhi | BlockchainCon India is the premier conference in India that brings together Developers, Experts, Researchers, Government, Enthusiasts and Companies dealing in Blockchain Technologies. |
```

Comment:
From the website, this is called `BlockchainCon` and not `Global Blockchain Conference`. Can you update it to match `BlockchainCon`?